mm2src/mm2_main/src/rpc/lp_commands/one_inch/types.rs

```diff
@@ -229,13 +229,13 @@ pub struct TxFields {
 impl TxFields {
     pub(crate) fn from_api_tx_fields(
         tx_fields: one_inch_api::classic_swap_types::TxFields,
-        dst_decimals: u8,
+        src_decimals: u8,
     ) -> MmResult<Self, FromApiValueError> {
         Ok(Self {
             from: tx_fields.from,
             to: tx_fields.to,
             data: BytesJson::from(hex::decode(str_strip_0x!(tx_fields.data.as_str()))?),
-            value: u256_to_big_decimal(U256::from_dec_str(&tx_fields.value)?, dst_decimals)?,
+            value: u256_to_big_decimal(U256::from_dec_str(&tx_fields.value)?, src_decimals)?,
```

Comment:
https://portal.1inch.dev/documentation/apis/swap/classic-swap/Parameter%20Descriptions/swap_params

this doc says that `value` is in `ETH` (but i agree this doesn't make a lot of sense if the src nor dst are `ETH`). am i looking at the wrong doc?

Reply:
In fact they accept and return all amounts in wei.
(looks like the doc, you refer to, also says about wei)

Reply:
We should just convert this value to ETH indeed, fixed, tyvm.